plugins/delphes/examples/EDM4Hep2LCIO_example.py

```diff
@@ -45,22 +54,32 @@ def generateEvents(inputFileName, outputFileName, nEvents):
         tracks = IMPL.LCCollectionVec(EVENT.LCIO.TRACK)
         recops = IMPL.LCCollectionVec(EVENT.LCIO.RECONSTRUCTEDPARTICLE)
 
+        #print ('nreco part  ',e.ReconstructedParticles.size(),'  ntracks  ',e.EFlowTrack.size())
```

Comment:
Please remove the commented print statements in this file.

Reply:
done